tests/phpunit/tests/rest-api/rest-request.php

```diff
@@ -650,4 +650,57 @@ public function test_set_param_follows_parameter_order() {
 			$request->get_json_params()
 		);
 	}
+
```

Comment:
We should add the ticket number as PHP docs. So

```
@ticket 40838
```

Reply:
thanks